### ISSUE TITLE
Integrating Razzle with existing react app

### ISSUE THREAD

**Issue by therise3107** [jaredpalmer/razzle#1099]
### Objective

Hey, we are trying to port our large scale Client-side app to SSR, We were exploring Next.js and Razzle and for our current case, we are choosing razzle.

Now, the problem is we cannot start with scratch and we certainly cannot use next.js unless we have no other option(using express) so I'm looking any possible solution or anyone who has already done this if possible.

### Outcome

I have tried individually installing razzle and express then modifying the webpack accordingly but we are stuck with node errors primarily relating to running client-side code on nodejs like undefined document and windows. Tried conditional but node_modules are using windows.

### Stack

Our stack is webpack for tooling, jss for styling, apollo v1.0 for API and React
Thanks in advance

**Comment by therise3107** [comment]
Closing this^ got it working
